source/MaterialXContrib/MaterialXMaya/CreateMaterialXNodeCmd.h

```diff
@@ -32,17 +32,25 @@ class CreateMaterialXNodeCmd : MPxCommand
     static MString NAME;
 
   private:
+    /// Specifies the type of shading node to create
+    enum class CreateAs
```

Comment:
This just sounds strange to me to be an enumeration name as it's not a noun (an adverb if my grammar is correct?)  Would  `CreationType` or `NodeType` be better here?

Reply:
Renamed.